examples/features/advancedtls/creds/localhost-openssl.cnf

```diff
@@ -0,0 +1,24 @@
+[req]
```

Comment:
Should these `cnf` files have comments to explain anything that isn't obvious, or point out what's important / not boilerplate?

Reply:
Discussed, no because they are out of scope for this as an example - they are a prerequisite for getting the credentials needed